pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -529,7 +529,7 @@ func (e *TokensController) generateTokenIfNeeded(serviceAccount *v1.ServiceAccou
 	if liveSecret.ResourceVersion != cachedSecret.ResourceVersion {
 		// our view of the secret is not up to date
 		// we'll get notified of an update event later and get to try again
-		klog.V(2).Infof("secret %s/%s is not up to date, skipping token population", liveSecret.Namespace, liveSecret.Name)
+		klog.V(2).InfoS("secret is not up to date, skipping token population", klog.KObj(liveSecret))
```

Comment:
Need to update according to the format of InfoS and capitalize the first letter

Reply:
done